### PR TITLE
Drop Node 10 support in blueprint engine spec

### PR DESCRIPTION
**Pull request by elwayman02**
This appears to have been missed in #9530

cc: @rwjblue

### CODE REVIEW

File: blueprints/app/files/package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "qunit-dom": "^1.6.0"
   },
   "engines": {
-    "node": "10.* || >= 12"
+    "node": ">= 12"
```
Comment (rwjblue) [inline blueprints/app/files/package.json:68]: Should be 
```suggestion
    "node": "12.* || 14.* || >= 16"
```

**Review by rwjblue — CHANGES_REQUESTED** [review]

**Comment by elwayman02** [comment]
@rwjblue I updated the fixtures to match the new engine support you suggested. Thanks!

**Comment by elwayman02** [comment]
@rwjblue are you able to restart these builds? The failure looks like a flaky test, but I don't have access to re-run them.

**Comment by rwjblue** [comment]
Restarted

**Comment by elwayman02** [comment]
Failures appear flaky again @rwjblue :(

**Comment by rwjblue** [comment]
The Embroider failures are:

```
not ok 1 Chrome 91.0 - [undefined ms] - Global error: Uncaught TypeError: Cannot read property 'prototype' of undefined at webpack://some-cool-app/../node-E3gpVVB0.tmp/node_modules/@ember-data/debug/setup.js?, line 20
    ---
        browser log: |
            testContext: [object Object]
            ERROR: Uncaught TypeError: Cannot read property 'prototype' of undefined at webpack://some-cool-app/../node-E3gpVVB0.tmp/node_modules/@ember-data/debug/setup.js?, line 20
            
    ...

```

Which is what @thoov reported in https://github.com/embroider-build/embroider/issues/826, definitely unrelated to these changes.

**Review by 1atAlcone — APPROVED** [review]
### - [ ] -

**Review by 1atAlcone — APPROVED** [review]

**Comment by 1atAlcone** [comment]
_hi 👋_

**Pull request merged**